lib/linux_admin/network_interface.rb

```diff
@@ -148,7 +148,7 @@ def start
     #
     # @return [Boolean] whether the command succeeded or not
     def stop
-      Common.run(Common.cmd("ifdown"), :params => [@interface]).success?
+      Common.run(Common.cmd("ifconfig"), :params => [@interface, "down"]).success?
```

Comment:
Additionally, while testing this I found that `ifconfig eth0 down` didn't actually bring the network interface down.

A subsequent call to `ifup eth0` failed because the interface was still active.